vllm/engine/arg_utils.py

```diff
@@ -868,6 +870,20 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             help="Override or set the pooling method in the embedding model. "
             "e.g. {\"pooling_type\": \"mean\", \"normalize\": false}.'")
 
+        parser.add_argument('--compilation-config',
+                            '-O',
+                            type=CompilationConfig.from_cli,
+                            default=None,
+                            help='torch.compile configuration for the model.'
+                            'When it is a number (0, 1, 2, 3), it will be '
+                            'interpreted as the optimization level.\n'
+                            'NOTE: level 0 is the default level without '
+                            'any optimization. level 1 and 2 are for internal '
+                            'testing only. level 3 is the recommended level '
```

Comment:
Maybe add that this (levels 1 and 2) could change in the future - if we actually have multiple optimization levels, we can use O1 and O2 and move the internal levels somewhere else

Reply:
our optimization is actually not well graded as levels. they appear as complicated configs. I would keep only levels 0/1/2/3 , and recommend using 3 for users.